Description: Script is trying to add third_party directory to sys.path.
 This package doesn't contain third_party modules, all third_party modules
 available as Debian packages and this package already have them in Depends.

--- a/ycmd/server_utils.py
+++ b/ycmd/server_utils.py
@@ -33,9 +33,6 @@
 def SetUpPythonPath():
   sys.path.insert( 0, os.path.join( DirectoryOfThisScript(), '..' ) )
 
-  from ycmd import utils
-  utils.AddNearestThirdPartyFoldersToSysPath( __file__ )
-
 
 def ExpectedCoreVersion():
   return int( open( os.path.join( DirectoryOfThisScript(), '..',
